### ISSUE TITLE
Macros in mongo db collection

### ISSUE THREAD

**Issue by starking8b** [syslog-ng/syslog-ng#3570]
could u please add macros to create collection in mongo db

**Comment by MrAnno** [comment]
This feature request was probably opened after a short discussion on Stack Overflow.

More info:
https://stackoverflow.com/questions/66154248/customizing-log-destination-syslog-ng-using-macros

```
mongodb(
  uri("mongodb://localhost:27017/syslog")
  collection("${HOST}${YEAR}${MONTH}${DAY}")    
);
```

**Comment by MrAnno** [comment]
#3621 adds template support for the `collection()` option.
This will most likely be included in syslog-ng v3.32.
